docs/components/confirm.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Confirm
+layout: page.jade
+sidebar: true
+collection: docs
+priority: 3
+path: confirm
+section: js
+---
+
+# Confirm
+<p class="lead">
+  A confirm is used to present a question messages to users' actions.
+</p>
+
+## Summary
+1. [Usage](#usage)
+2. [Options](#options)
+3. [Size option](#size-option)
+
+
+## Usage
+The confirm component is an empty container where you can add a message that will be presented to users and receives an answer about something. It has an Ok and Cancel buttons by default
```

Comment:
Que tal depois de `users` deixar algo assim: `... so they can confirm or not the execution of an specific action.`
Dae pode omitir essa parte sobre os botões.